pygeos/test/test_misc.py

```diff
@@ -71,6 +71,32 @@ def foo():
         foo()
 
 
+@pytest.mark.parametrize("version", ["3.6.0", "3.8.0"])
+def test_requires_geos_adapts_docstring(version, mocked_geos_version):
+    """The requires_geos decorator adapts the docstring."""
+
+    @requires_geos(version)
+    def foo():
```

Comment:
You could also put this function just before the test function so it's not nested, and then you don't have that strange indentation issue to comment about.